ceph.spec.in

```diff
@@ -255,6 +255,8 @@ Requires:      lsb-release
 %if 0%{?fedora} || 0%{?rhel}
 Requires:      redhat-lsb-core
 %endif
+Conflicts:      ceph < %{epoch}:10
+Obsoletes:      ceph < %{epoch}:10
```

Comment:
I don't think we need the Conflicts line here (and in all the other cases). The Conflicts line would probably only cause problems for us.

As for the Obsoletes line that does make sense, not sure if it is actually necessary though as this might already be covered by the tight 'Requires: <exact_version>' lines.